core/src/replay_stage.rs

```diff
@@ -329,6 +377,26 @@ impl ReplayStage {
 
         trace!("replay stage");
         // Start the replay stage loop
+
+        let (cost_update_sender, cost_update_receiver): (
+            Sender<ExecuteTimings>,
+            Receiver<ExecuteTimings>,
+        ) = channel();
+
+        let exit_clone = exit.clone();
+        let blockstore_clone = blockstore.clone();
+        let t_service = Builder::new()
```

Comment:
nit: `t_service -> t_cost_model_service`

Reply:
Thought this thread to do all sort of background 'services', but in reality all it does is cost_model stuff. Yeah, name it to be specific is good idea.